src/main/java/org/fcrepo/upgrade/utils/F47ToF5UpgradeManager.java

```diff
@@ -297,6 +326,78 @@ private void upgradeRdfAndCreateHeaders(final TemporalAccessor versionTimestamp,
         LOGGER.debug("containerSubject={}", containerSubject);
     }
 
+    private void convertAcl(final Path convertedProtectedResourceLocation, String protectedResource, String aclUri) {
+        //locate the exported acl rdf on disk based on aclURI
+        final var relativeAclPath = create(aclUri).getPath();
+        final var aclDirectory = Path.of(this.config.getInputDir().toPath().toString(), relativeAclPath.toString());
+        final var aclRdfFilePath = aclDirectory.toString() + TURTLE_EXTENSION;
+        final var newAclResource = ResourceFactory.createResource(protectedResource + "/fcr:acl");
+        final var aclModel = createModelFromFile(Path.of(aclRdfFilePath));
+        final var aclTriples = new ArrayList<Statement>();
+        final var allowablePredicates = Arrays.asList(RDF.type, FEDORA_CREATED_DATE, FEDORA_LAST_MODIFIED_DATE, FEDORA_CREATED_BY, FEDORA_LAST_MODIFIED_BY);
+        aclModel.listStatements().toList().stream().filter(x->allowablePredicates.contains(x.getPredicate()))
+                .forEach(x -> {
+            final var obj = x.getObject();
+            final var predicate = x.getPredicate();
+            if ( !(predicate.equals(RDF.type) && obj.isResource() && obj.equals(ACL))) {
+                aclTriples.add(aclModel.createStatement(newAclResource, x.getPredicate(), x.getObject()));
```

Comment:
May as well use the variables created above:
- `predicate` instead of `x.getPredicate()`
- `obj` instead of `x.getObject()`